vars/uploadToS3.groovy

```diff
@@ -1,7 +1,8 @@
 void call(Map args = [:]){
 
-    withAWS(role: 'opensearch-bundle', roleAccount: "${AWS_ACCOUNT_PUBLIC}", duration: 900, roleSessionName: 'jenkins-session') {
-        s3Upload(file: args.sourcePath, bucket: args.bucket, path: args.path)
+    withCredentials([string(credentialsId: 'AWS_ACCOUNT_PUBLIC', variable: 'aws_account_public')]) {
```

Comment:
`credentialsId` values should be start with `jenkins-*` prefix, this way we can configure the jenkins master role to only pull secrets that has `jenkins-*` prefix Example as https://github.com/opensearch-project/opensearch-build/blob/main/vars/copyContainer.groovy#L25

Reply:
I believe whatever secrets we store in under the account will all belong to jenkins. So won't be it an overhead?

Reply:
It will also contain secrets that are part of `OpenSearch-CI-Config-Dev `stack
https://github.com/opensearch-project/opensearch-ci, also its possible to to filter with `jenkins-*` prefix
<img width="1134" alt="Screen Shot 2022-06-22 at 5 43 13 PM" src="https://user-images.githubusercontent.com/28733332/175144000-d52fe343-7246-4629-804f-f311884276a9.png">

Reply:
True but credentials are only fetched using a particular tag as per the plugin documentation. Since we are not adding these tags in config or CI stacks, they will be on jenkin's credential DB. 
https://plugins.jenkins.io/aws-secrets-manager-credentials-provider/#plugin-content-usage

Reply:
I just want to eliminate jenkins master able to query all available secret manager credentials in that account, would be good to have only credentials with prefix `jenkins-*` be queried, then it would be jenkins app specific.

Reply:
I see. what about existing secrets. Looks like we cannot change name of already stored secrets.

Reply:
Ya we have to recreate a new one and change the value in jenkinsfile, @peterzhuamazon thoughts please?

Reply:
Ya we have to recreate a new one and change the value in jenkinsfile, @peterzhuamazon thoughts please?